src/PointwiseFunctions/GeneralRelativity/GeodesicAcceleration.hpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace gr {
+/// @{
+
+/*!
+ * \brief Computes the coordinate geodesic acceleration in the inertial frame.
+ *
+ * \details The geodesic acceleration in coordinate form is given by
+ * \f{equation}
+ *  \frac{d^2 x^i}{d t^2} = (v^i \Gamma^0_{00} - \Gamma^i_{00} )  + 2 v^j (v^i
+ * \Gamma^0_{j0} - \Gamma^i_{j0} ) + v^j v^k (v^i \Gamma^0_{jk} - \Gamma^i_{jk}
+ * ), \f}
+ * where \f$v^i\f$ is the coordinate velocity, \f$\Gamma^\mu_{\nu \rho}\f$ are
+ * the spacetime Christoffel symbols of the second kind, and all latin indices
+ * are spatial.
+ */
+template <size_t Dim>
+void geodesic_acceleration(
```

Comment:
Yeah I can't think of another place for this. However, since these functions currently only act on `double`s, they aren't really "pointwise". Could you template them on the `DataType` and then instantiate them for `double` and `DataVector` in the `cpp`?

For the test, you can keep the conserved quantities test using `double` (since Boost and DV are annoying together), but have the circular orbit test check both `double` and `DataVector`.